test/integration/merkle-streamer/factory/create-merkle-streamer-ll/createMerkleStreamerLL.t.sol

```diff
@@ -12,26 +14,47 @@ contract CreateMerkleStreamerLL_Integration_Test is MerkleStreamer_Integration_T
         MerkleStreamer_Integration_Test.setUp();
     }
 
+    function test_RevertWhen_CampaignNameTooLong() external {
+        MerkleStreamer.ConstructorParams memory params = defaults.constructorParams();
+        LockupLinear.Durations memory streamDurations = defaults.durations();
+        string memory ipfsCID = defaults.IPFS_CID();
+        uint256 aggregateAmount = defaults.AGGREGATE_AMOUNT();
+        uint256 recipientsCount = defaults.RECIPIENTS_COUNT();
+
+        params.name = "this string is longer than 32 characters";
+
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                Errors.SablierV2MerkleStreamerFactory_CampaignNameTooLong.selector, bytes(params.name).length, 32
+            )
+        );
```

Comment:
We should also write a fuzz test that fuzzes the `name`.

Or maybe we can incorporate this fuzzing in the existing `testFuzz_CreateMerkleStreamerLL` test?

Reply:
Interesting, but why do you want to fuzz `name`? We have an [integration test for name length](https://github.com/sablier-labs/v2-periphery/blob/af89263a36675a30efb1bcb2e2ec14a27c25003c/test/integration/merkle-streamer/factory/create-merkle-streamer-ll/createMerkleStreamerLL.t.sol#L17C1-L40C6) and since it is constant, as long as it fits into 32 bytes, there can never be a problem.